test/placement/placement_test.cpp

```diff
@@ -30,9 +47,37 @@ TEST_CASE_METHOD(NetlistFixture, "Placement: placing a cell", "[placement]") {
     ophidian::util::Location cell2Location(20, 10);
     placement.placeCell(cell2, cell2Location);
 
-    ophidian::util::Location placedCell1Location = placement.location(cell1);
-    ophidian::util::Location placedCell2Location = placement.location(cell2);
+    ophidian::util::Location placedCell1Location = placement.cellLocation(cell1);
+    ophidian::util::Location placedCell2Location = placement.cellLocation(cell2);
     REQUIRE(cell1Location == placedCell1Location);
     REQUIRE(cell2Location == placedCell2Location);
     REQUIRE(!(placedCell1Location == placedCell2Location));
 }
+
+TEST_CASE_METHOD(NetlistFixture, "Placement: placing an input pad", "[placement]") {
```

Comment:
This code duplication doesn't smell good for me.

Why not just using the **old approach**?:
- Consider the position of a pin a Derived Property (Its derived because its obtained by summing the position of its parent cell and the library offset).
- A pad is a special `StandardCell::Pin` without owner `Cell` and its offset is equals to its actual position.
- A pad has a correspondent `Netlist::Pin` which should be mapped to the respective `StandardCell::Pin`. Since the `Netlist::Pin` doesn't have a owner `Netlist::Cell`, its position is just the offest, obtained from the library.